parser/parser.go

```diff
@@ -85,7 +91,14 @@ func (p *Parser) Parse(source common.Source) (*exprpb.ParsedExpr, *common.Errors
 	if !ok {
 		buf = runes.NewBuffer(source.Content())
 	}
-	e := impl.parse(buf, source.Description())
+	var e *exprpb.Expr
+	if buf.Len() > p.expressionSizeCodePointLimit {
+		e = impl.reportError(common.NoLocation,
+			"expression size exceeds limit: size: %d, limit %d",
```

Comment:
Maybe say `codepoint limit`? I think I missed this on the Java review.

Reply:
Done.